Casks/e/eid-mw.rb

```diff
@@ -0,0 +1,30 @@
+cask "eid-mw" do
+  version "5.0.26"
+  sha256 "61312109f3e9525f95f47cf171eeec84c08472b7d898e71ee20737202c3ad592"
+
+  url "https://eid.belgium.be/sites/default/files/software/eID-Quickinstaller-#{version}.dmg"
+  name "Belgian EID Middleware"
+  desc "E-ID Middleware is the middleware for the Belgian e-id system"
+  homepage "https://eid.belgium.be/"
+
+  livecheck do
+    url "https://eid.belgium.be/en"
+    regex(/href=.*?eID[-_]?Quickinstaller[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
+
+    strategy :page_match do |page, regex|
+      download_id = page[%r{element_os-mac-os.*/en/download/(\d+)/license.*?download\s+eID\s+quickinstaller}im, 1]
```

Comment:
```suggestion
    regex(/href=.*?eID(?:(?:%20|\s)+|[._-])?Quickinstaller[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
    strategy :page_match do |page, regex|
      # The first download button on the site is expected to correspond to the
      # eID-Quickinstaller dmg used in this cask.
      download_id = page[%r{element_os-mac-os.*?/download/(\d+)/license}im, 1]
```

This `livecheck` block isn't working because the regex in the `strategy` block for identifying the download ID is trying to match something on the page that doesn't exist. There is no `quickinstaller` text on the page and I don't think there's any surrounding text that we can use to ensure that the regex only matches the Quickinstaller download, so instead we simply have to assume that the first `/download/.../license` URL on the page is the Quickinstaller download. This is currently the case and may not always remain true but I don't think there's an alternative, so this includes a comment to explain this shortcoming.

Reply:
Ok. Well I guess we will notice if/when it breaks. Kind of sad they don't maintain versioning on their github. Must admit that I didn't fully understand what that regex was doing yet...